native_client/deepspeech.h

```diff
@@ -8,24 +8,14 @@
 namespace DeepSpeech
 {
 
-  class Private;
+  struct Private;
+
+  struct StreamingState;
 
   class Model {
     private:
       Private* mPriv;
```

Comment:
Classes that pass over the ABI boundary should have a non-inline virtual destructor as far as I know.

Reply:
I assume you're talking about `Model`, not `Private`, as that's an opaque pointer. I made `Model`'s destructor virtual.